assignment3.txt

```diff
@@ -0,0 +1,7 @@
+1. The code is not valid as the syntax for the for loop is incorrect. First, i is never declared before it is used and is not given a starting value. Inside the loop, there is also no integer in the printf statement. 
+
+2. The variable i should not be reused within the loop because the computer will not be able to differentiate between the i being used to increment in the loop and the i in loop.
```

Comment:
that's not true - however, this will cause the code to enter an infinite loop. Also, there shouldn't be a semi-colon after the second parenthesis.